src/pyhf/tensor/numpy_backend.py

```diff
@@ -297,15 +341,18 @@ def percentile(self, tensor_in, q, axis=None, interpolation="linear"):
             NumPy ndarray: The value of the :math:`q`-th percentile of the tensor along the specified axis.
 
         """
-        return np.percentile(tensor_in, q, axis=axis, interpolation=interpolation)
+        # see numpy/numpy#22125
```

Comment:
Looking at the issue, is switching from `interpolation` to `method` an option? It seems to only be available starting from `numpy` 1.22 though.

Reply:
We would need to update minimum bounds for this, so yeah. It would be nice, but numpy 1.22 is a relatively stricter lower bound (and lcgenv doesn't seem to have that when I looked)

Reply:
@alexander-held I opened up Issue #1772 for this back in February 2022. I agree it would be nice to have though.